app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -16,7 +16,7 @@ $after-link-padding: govuk-spacing(4);
 $after-button-padding-right: govuk-spacing(4);
 $after-button-padding-left: govuk-spacing(4);
 
-$search-button-padding-top-bottom: govuk-spacing(3);
+$search-button-padding-top-bottom: govuk-spacing(4);
```

Comment:
The padding now seems to be the same on all sides - top, right, bottom, and left. Should we update the variable name to `search-button-padding`?

Reply:
Good spot, I've updated the variable name to `search-button-padding` as suggested, thanks 👍